File: src/main/java/com/gmail/borlandlp/minigamesdtools/geoip/GeoData.java
package com.gmail.borlandlp.minigamesdtools.geoip;

import java.util.ArrayList;
import java.util.List;

public class GeoData {
    public String ip;
    public String type; // ipv4, ipv6
    public String continentCode; // Europe..
    public String continentName;
    public String countryCode; // EN, RU..
    public String countryName; // United Kingdom, Russia..

    public String locGeonameId;
    public String locCapital; // London, Moscow..

    public List<LangData> languages = new ArrayList<>();
}

File: src/main/kotlin/com/gmail/borlandlp/minigamesdtools/geoip/GeoData.kt
package com.gmail.borlandlp.minigamesdtools.geoip

object GeoData {
    @JvmField
    var ip: String? = null
    @JvmField
    var type // ipv4, ipv6
            : String? = null
    @JvmField
    var continentCode // Europe..
            : String? = null
    @JvmField
    var continentName: String? = null
    @JvmField
    var countryCode // EN, RU..
            : String? = null
    var countryName // United Kingdom, Russia..
            : String? = null
    @JvmField
    var locGeonameId: String? = null
    @JvmField
    var locCapital // London, Moscow..
            : String? = null
    @JvmField
    var languages: MutableList<LangData> = mutableListOf()
}